### PR TITLE
[20.05] Mulled packaging dependency fixes

### PR DESCRIPTION
**Pull request by mvdbeek**
So `pip install galaxy-tool-util[mulled]` will give you functional mulled commands. xref https://github.com/galaxyproject/galaxy-lib/issues/156

### CODE REVIEW

**Comment by bgruening** [comment]
It also uses the `conda` package, the API at the moment. I needed to add this as well. But we can fix this on the conda side.

**Comment by nsoranzo** [comment]
@mvdbeek Can you please try to remove all new and old symlinks to individual test files in `packages/tool-util/tests/` (but keep `__init__.py`) and add a single symlink: `packages/tool_util/tests/tool_util` -> `../../../test/unit/tool_util/`

File: packages/tool_util/setup.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def get_var(var_name):
     include_package_data=True,
     install_requires=requirements,
     extras_require={
-        'condatesting': ['jinja2'],
+        'mulled': ['conda', 'cytoolz', 'jinja2', 'Whoosh'],
```
Comment (nsoranzo) [inline packages/tool_util/setup.py]: What's cytoolz for? I couldn't find a reference in the repo (on the phone).

**Review by nsoranzo — APPROVED** [review]

File: packages/tool_util/setup.py (hunk 1 above)
Comment (mvdbeek) [reply packages/tool_util/setup.py]: cytoolz is an undeclared dependency of the conda package on pypi.

**Pull request merged**